scripts/commit-shiki-update.sh

```
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+if ! git diff --exit-code package.json > /dev/null; then
+  echo "Shiki version changed, building new release"
+  shiki_version=$(node -p "require('./package.json').devDependencies.shiki.replace(/[^0-9.]/g, '')")
+  read major minor patch <<< $(tr . " " <<< "$shiki_version")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)